src/main/java/net/liquidpineapple/pang/objects/Player.java

```diff
@@ -77,5 +80,31 @@ public void doUpdate() {
                 Application.getBoard().addObject(newRope);
             }
         }
+
+        if(collisionPlayer()){
+            if(!this.isHit) {
+                Application.lifeKeeper.loseLife();
+                this.isHit = true;
+            }
```

Comment:
These lines are all references to static methods and fields. It should thus be done like this:

```java
if(!isHit) {
    LifeSystem.loseLife();
    isHit = true;
}
```